samples/v1.0/Elements/Image.BackgroundColor.json

```diff
@@ -5,13 +5,13 @@
 	"body": [
 		{
 			"type": "Image",
-			"url": "http://messagecardplayground.azurewebsites.net/assets/circleontransparentbackground.png",
+			"url": "http://adaptivecards.io/content/cats/2.png",
 			"size": "small",
 			"backgroundColor": "#FF0000FF"
```

Comment:
First test image looks wrong; Blue background doesn't show up on first image and, color doesn't seems red on the people style image.

Reply:
Looks like someone added an additional background color test, so we now have both Tests\Image.BackgroundColor and v1.0\Elements\Image.Background color. I'll figure out where this came from and get us back down to one.

---
In reply to: [207621147](https://github.com/Microsoft/AdaptiveCards/pull/1765#discussion_r207621147) [](ancestors = 207621147)

Reply:
I moved the original file (with my changes to cats) out of the v1.0 folder and into the tests folder. 

---
In reply to: [207641398](https://github.com/Microsoft/AdaptiveCards/pull/1765#discussion_r207641398) [](ancestors = 207641398,207621147)